From 5a30409af1e760a04b822a46be68519ad6e4cb2d Mon Sep 17 00:00:00 2001
From: "Simental Magana, Marcos" <marcos.simental.magana@intel.com>
Date: Wed, 20 Sep 2017 09:59:59 -0500
Subject: [PATCH] install autostart in /usr/share rather than /usr/etc

---
 data/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/CMakeLists.txt b/data/CMakeLists.txt
index e41dddc..4be6810 100644
--- a/data/CMakeLists.txt
+++ b/data/CMakeLists.txt
@@ -1,5 +1,5 @@
 set(filedeps)
-set(autostartdir ${SYSCONF_INSTALL_DIR}/xdg/autostart)
+set(autostartdir ${SHARE_INSTALL_PREFIX}/xdg/autostart)
 set(desktopdir ${SHARE_INSTALL_PREFIX}/applications)
 set(themedir ${privdatadir}/theme)
 
-- 
2.14.1

