--- evolution-3.24.1/cmake/modules/SetupBuildFlags.cmake~	2017-04-10 08:02:00.000000000 +0000
+++ evolution-3.24.1/cmake/modules/SetupBuildFlags.cmake	2017-04-10 20:55:17.052260164 +0000
@@ -10,7 +10,6 @@
 
 macro(setup_build_flags _maintainer_mode)
 	list(APPEND proposed_flags
-		-Werror-implicit-function-declaration
 		-Wformat
 		-Wformat-security
 		-Winit-self
--- evolution-3.24.1/cmake/modules/FindSMIME.cmake~	2017-04-10 08:02:00.000000000 +0000
+++ evolution-3.24.1/cmake/modules/FindSMIME.cmake	2017-04-11 00:22:32.408264032 +0000
@@ -74,10 +74,6 @@
 
 unset(CMAKE_REQUIRED_INCLUDES)
 
-if(NOT _have_headers)
-	message(FATAL_ERROR "NSPR headers not found. Use -DWITH_NSPR_INCLUDES=/path/to/nspr to specify the include dir of NSPR.")
-endif(NOT _have_headers)
-
 set(MANUAL_NSPR_INCLUDES "${WITH_NSPR_INCLUDES}")
 
 set(nsprlibs "-lplc4 -lplds4 -lnspr4")
--- evolution-3.24.1/cmake/modules/FindSMIME.cmake~	2017-04-11 00:22:32.000000000 +0000
+++ evolution-3.24.1/cmake/modules/FindSMIME.cmake	2017-04-11 00:28:48.829276237 +0000
@@ -109,10 +109,6 @@
 
 unset(CMAKE_REQUIRED_INCLUDES)
 
-if(NOT _have_headers)
-	message(FATAL_ERROR "NSS headers not found. Use -DWITH_NSS_INCLUDES=/path/to/nss to specify the include dir of NSS.")
-endif(NOT _have_headers)
-
 set(MANUAL_NSS_INCLUDES "${WITH_NSS_INCLUDES} ${MANUAL_NSPR_INCLUDES}")
 
 set(nsslibs "-lssl3 -lsmime3 -lnss3")
